cms/static/js/factories/textbooks.js

```diff
@@ -5,6 +5,7 @@ define([
     return function(textbooksJson) {
         var textbooks = new TextbookCollection(textbooksJson, {parse: true}),
             tbView = new ListTextbooksView({collection: textbooks});
+        window.models.TextbookCollection = textbooks;
```

Comment:
This is the ONE line interop warriors will need to add into legacy code. All you gotta do is set a global reference to the Backbone collection you wish to listen to as soon as it's initialized. Let's use `window.models` as a home for these. Not super elegant to do it global like this, but the alternative involves a lot more legacy-munging.

Reply:
👏 👏 👏